test/labeling/model/test_label_model.py

```diff
@@ -17,36 +16,40 @@ def setUpClass(cls):
         cls.m = 10
         cls.k = 2
 
-    def _test_label_model(self, data, test_acc=True):
-        label_model = LabelModel(k=data.k, verbose=False)
-        label_model.train_model(
-            data.L,
-            deps=data.E,
-            class_balance=data.p,
-            n_epochs=1000,
-            log_train_every=200,
+    def test_model_init(self):
+        label_model = LabelModel(k=2, verbose=False)
+
+        # Test class balance
+        Y_dev = np.array([1, 1, 2, 2, 1, 1, 1, 1, 2, 2])
+        label_model._set_class_balance(class_balance=None, Y_dev=Y_dev)
```

Comment:
Not to pontificate, but unit testing internal state via private methods is contentious: https://stackoverflow.com/questions/34571/how-do-i-test-a-private-function-or-a-class-that-has-private-methods-fields-or

It might be ok to do this here. Honestly, these methods can likely be made public

Reply:
I think making these methods public is the way to go - everything is private except train, which makes it hard to test